scripts/export_event_types.py

```diff
@@ -127,6 +91,10 @@
                 return_partition=True,
             )
 
+            import pickle
```

Comment:
I don't mind using pickle here of course, but notice that everywhere else we used joblib. Is there a good reason to use pickle here instead?

Reply:
I'm sorry, this was just a test I did to check the partitions, but it shouldn't have been commited in the code, my bad.